structs.h

```diff
@@ -602,7 +602,7 @@ typedef struct MonsterStruct { // note: missing field _mAFNum
 	unsigned char leader;
 	unsigned char leaderflag;
 	unsigned char packsize;
-	unsigned char mlid;

```

Comment:
There are 200 monsters and in Hellfire, all monsters can receive light (if baserked). To avoid having to deal with the save game compatibility of expanding this to `Sint16`/`Sint32` I recommend that we set `NO_LIGHT` to be `UCHAR_MAX`. This may in fact even be bin exact.

Reply:
I believe I've tried that and it didn't work, light code takes an int which is -1, you can't play around that with 255

Reply:
You're also right that I got this wrong. I thought it was the light tracking the monster id, but it's the other way around so a signed char is plenty big.